packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -4925,6 +4968,8 @@ class SemanticsConfiguration {
       .._isMergingSemanticsOfDescendants = _isMergingSemanticsOfDescendants
       .._textDirection = _textDirection
       .._sortKey = _sortKey
+      .._identifier = _identifier
+      ..identifier = _identifier
```

Comment:
duplicated line?

Reply:
good catch. Will fix, thanks.

Reply:
fixed in 0b98178af46c82d48dd6e008290e23f8a21f122a